From haver@vnet.ibm.com Wed Jan 17 10:38:15 2007
From: Frank Haverkamp <haver@vnet.ibm.com>
Date: Wed, 17 Jan 2007 17:51:18 +0100
Subject: SYSFS: Fix missing include of list.h in sysfs.h
Cc: greg@kroah.com
Message-ID: <1169052679.21717.9.camel@localhost.localdomain>


Sysfs.h uses definitions (e.g. struct list_head s_sibling) from list.h
but does not include it.

Signed-off-by: Frank Haverkamp <haver@vnet.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/sysfs.h |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/include/linux/sysfs.h
+++ gregkh-2.6/include/linux/sysfs.h
@@ -11,6 +11,7 @@
 #define _SYSFS_H_
 
 #include <linux/compiler.h>
+#include <linux/list.h>
 #include <asm/atomic.h>
 
 struct kobject;
